### ISSUE TITLE
Use system-installed modules

### ISSUE THREAD

**Issue by habtie-phys** [fortran-lang/fpm#851]
I am beginner in FORTRAN programming. I found fpm very easy and friendly to build FORTRAN projects. But I am having a hard time of linking my project to externally installed modules. The fpm documentation mentioned the linking procedure only briefly without an example .  Could someone please help me translating the following standard terminal build command to fpm build command?

`gfortran  main.f90  -o  main  -I/usr/local/include  -L/usr/local/lib  -lnetcdff`

In short, the `main.f90` file uses a netcdf module installed in `usr/local` folder. 

Thank you for your help.
